### DISCUSSION TITLE
What are keys in React?

### DISCUSSION THREAD

**Discussion by Akash2377** [Akash2377/Akash2377_portfolio#9] [Q&A] [answered]
What are keys in React?

**Comment by Manu2807** [comment, marked as answer]
A key is a unique identifier. In React, it is used to identify which items have changed, updated, or deleted from the Lists. It is useful when we dynamically created components or when the users alter the lists.

**Comment by Manu2807** [comment]
A key is a unique identifier. In React, it is used to identify which items have changed, updated, or deleted from the Lists. It is useful when we dynamically created components or when the users alter the lists.
